.eslintrc

```diff
@@ -14,3 +14,13 @@ plugins:
 rules:
   no-console: 0  # This project purposely uses console.
   prefer-reflect: 0
+  jsdoc/check-param-names: 1
```

Comment:
you might want to add these rules to https://github.com/dcentralize/eslint-config-d-centralize ?